File: LondonMeetup/app/src/main/java/com/suyashsrijan/londonmeetup/API/tfl/enums/Stoptypes.java
package com.suyashsrijan.londonmeetup.API.tfl.enums;

public enum Stoptypes {
    PUBLIC_BUS_COACH_TRAM {
        public String toString() {
            return "NaptanPublicBusCoachTram";
        }
    },

    METRO_STATION {
        public String toString() {
            return "NaptanMetroStation";
        }
    },

    BIKE_STOP {
        public String toString() {
            return "BikeStop";
        }
    }
}

File: LondonMeetup_v2/app/src/main/java/com/suyashsrijan/londonmeetup/oldcode/API/tfl/enums/Stoptypes.kt
package com.suyashsrijan.londonmeetup.API.tfl.enums

enum class Stoptypes {
    PUBLIC_BUS_COACH_TRAM {
        override fun toString(): String {
            return "NaptanPublicBusCoachTram"
        }
    },

    METRO_STATION {
        override fun toString(): String {
            return "NaptanMetroStation"
        }
    },

    BIKE_STOP {
        override fun toString(): String {
            return "BikeStop"
        }
    }
}
